### PR TITLE
Temporarily disable an ApiControllerTest on CircleCI after Ubuntu 14 upgrade

### PR DESCRIPTION
**Pull request by davidsbailey**
example failure after upgrading our CircleCI project to Ubuntu 14: https://circleci.com/gh/code-dot-org/code-dot-org/31201

### CODE REVIEW

File: dashboard/test/controllers/api_controller_test.rb [hunk 1]
```diff
@@ -245,6 +245,8 @@ def make_progress_in_section(script)
   end
 
   test "should get surveys for section with script with anonymous level_group assessment" do
+    skip "temporarily disabled on CircleCI after Ubuntu 14 upgrade" if ENV['CI']
```
Comment (ashercodeorg) [inline dashboard/test/controllers/api_controller_test.rb:248]: It looks like the failure is the result of array elements being reordered. Is order important (in which case why are we disabling a failing test)? Is order unimportant (in which case why don't we simply permute the elements in the expected response)?

**Comment by ashercodeorg** [comment]
That said, LGTM, assuming that active investigation is happening.

**Comment by davidsbailey** [comment]
Thanks Asher! To summarize offline discussion, (1) this is only being disabled on circle and (2) the plan is to temporarily disable this to get circle green again, and then investigate fixing this failure so the test can be re-enabled on circle.

**Comment by asherkach** [comment]
(see also #14201)

**Pull request merged**